libs/components/data-manager/src/lib/modules/data-manager/data-manager-column-picker/data-manager-column-picker.component.html

```diff
@@ -25,9 +26,17 @@
     </sky-data-manager>
   </sky-modal-content>
   <sky-modal-footer>
+    <sky-alert
+      *ngIf="!isAnyDisplayedColumnSelected"
+      alertType="danger"
+      [closeable]="false"
+    >
+      Select at least one column.
```

Comment:
This string needs to be in a resource file and displayed using the `skyLibResources` pipe. See other instances of that technique elsewhere in this file.

Reply:
resolved in new PR. closing this PR.